libcontainer/configs/cgroup.go

```diff
@@ -57,6 +57,11 @@ type Cgroup struct {
 	// MEM to use
 	CpusetMems string `json:"cpuset_mems"`
 
+	// Process limit; set to `0' to disable limit.
+	// While technically `0' is a valid PID limit, it does not make sense in the
+	// context of a container -- it is identical to a limit of `1'.
```

Comment:
@yangdongsheng But ... it doesn't actually have a practical meaning. Having a limit of `0` is **precisely** identical to having a limit of `1`. Both prevent `fork()` and `clone()` in a container with a single process in it. There's no special code path that the pids controller goes through if the limit is specifically `0` (ref: I wrote it).